api/migrations/0002_data_migration.py

```diff
@@ -89,8 +89,16 @@ def _write_languages_and_levels(self):
         )
         Language.objects.bulk_create(languages)
 
-        # no C2 at this school
-        levels = (Level(id=id_) for id_ in ("A0", "A1", "A2", "B1", "B2", "C1"))
+        levels = (
+            Level(id=id_) for id_ in (
```

Comment:
You are able to iterate just through enum class LanguageLevelId.

`Level(id=id_) for id_ in LanguageLevelId`

Reply:
Done